traitsui/testing/tester/locator.py

```diff
@@ -35,3 +35,17 @@ class TargetByName:
     """
     def __init__(self, name):
         self.name = name
+
+class DefaultTarget:
+    """ A locator for locating a default target.  In many cases, handlers may
```

Comment:
Nit:  "many" maybe an overstatement depending on future additions to the registry... maybe "some"?